### PR TITLE
Python Styleguide

### PR DESCRIPTION
**Pull request by jgjay**
Starting point for a Python styleguide, mainly referencing out to [PEP8](https://www.python.org/dev/peps/pep-0008/) and [The Elements of Python Style](https://github.com/amontalenti/elements-of-python-style) with a couple of additional points from discussions in `#python` on Slack.

@brunns @r4vi @andreagrandi

### CODE REVIEW

File: python.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Python styleguide
+
+Reccomended reading:
```
Comment (andreagrandi) [inline python.md]: isn't it ```recommended``` ? (I may be wrong...not my main language :P )

File: python.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+* [PEP8 - Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/)
+* [The Elements of Python Style](https://github.com/amontalenti/elements-of-python-style)
+
+## Formatting
+
+Python has a strong and widely followed styleguide called [PEP8], you should follow [PEP8].
+
+If you want to add checking and enforcement to your project the reccomended tool is [flake8]. You may want to 
+[adjust the 79-character line length limit][flake8-config] with the `max-line-length` config option.
```
Comment (andreagrandi) [inline python.md]: I've been using **100** as length for a while, as a compromise, but I'm keen to change it if we all agree on a value.

**Comment by r4vi** [comment]
@andreagrandi  I don't think the guide is telling anyone to use a set number e.g 79 - it's being explicit that changing from 79 is fine

File: python.md (hunk 1 above)
Comment (jgjay) [inline python.md]: yes it is, I *always* get that wrong

**Comment by brunns** [comment]
Needs link added from `README.md`.

**Pull request merged**